app/main.py

```diff
@@ -1 +1,5 @@
-# write your code here
+def copy_file(command: str) -> None:
+    list_files = command.split()
```

Comment:
You should verify that the command string contains exactly three elements and starts with 'cp'. This is important to ensure the command is formatted correctly as per the task requirements.